### PR TITLE
feat: add variable to disable indexing

### PR DESCRIPTION
**Pull request by tkloht**
currently our robots directive always allows indexing in production, for our purposes we don't want the app indexed though.
this PR adds an option to allow indexing, to avoid a breaking change i'm making this `true` by default.

### CODE REVIEW

File: terraform-module/main.tf [hunk 1]
```diff
@@ -53,6 +53,7 @@ module "cdn" {
   domain_name   = local.domain_name
   bucket_name   = module.s3.bucket_name
   block_iframes = var.block_iframes
+  is_indexed    = var.is_indexed
```
Comment (binaryannie) [inline terraform-module/main.tf]: Would it make sense to add the word robots in there? Indexed can mean a lot of things

**Review by binaryannie — APPROVED** [review]
Thanks a lot for taking care of this @tkloht ❤️

File: terraform-module/main.tf (hunk 1 above)
Comment (tkloht) [reply terraform-module/main.tf]: true, renamed this to `is_robots_indexing_allowed`. a bit long but i think it should be easy to understand.

**Review by igorDolzh — APPROVED** [review]
Looks good!

**Pull request merged**